multiple-servers/README.md

```diff
@@ -0,0 +1,690 @@
+Complete requests:      5000
+Failed requests:        0
+Total transferred:      885000 bytes
+HTML transferred:       20000 bytes
+Requests per second:    2680.08 [#/sec] (mean)
+Time per request:       9.328 [ms] (mean)
+Time per request:       0.373 [ms] (mean, across all concurrent requests)
+Transfer rate:          463.26 [Kbytes/sec] received
+
+Connection Times (ms)
+              min  mean[+/-sd] median   max
+Connect:        0    1   2.0      1      59
+Processing:     1    8  24.2      3     322
+Waiting:        0    7  20.3      3     322
+Total:          1    9  24.3      4     323
+
+Percentage of the requests served within a certain time (ms)
+  50%      4
+  66%      4
+  75%      5
+  80%      6
+  90%     28
+  95%     34
+  98%     37
+  99%     62
+ 100%    323 (longest request)
+```
+
+And the static server:
+
+```console
+> ab -n 5000 -c 25 "http://127.0.0.1:8080/"
+This is ApacheBench, Version 2.3 <$Revision: 1901567 $>
+Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
+Licensed to The Apache Software Foundation, http://www.apache.org/
+
+Benchmarking 127.0.0.1 (be patient)
+Completed 500 requests
+Completed 1000 requests
+Completed 1500 requests
+Completed 2000 requests
+Completed 2500 requests
+Completed 3000 requests
+Completed 3500 requests
+Completed 4000 requests
+Completed 4500 requests
+Completed 5000 requests
+Finished 5000 requests
+
+
+Server Software:        nginx/1.23.1
+Server Hostname:        127.0.0.1
+Server Port:            8080
+
+Document Path:          /
+Document Length:        607 bytes
+
+Concurrency Level:      25
+Time taken for tests:   1.502 seconds
+Complete requests:      5000
+Failed requests:        0
+Total transferred:      4165000 bytes
+HTML transferred:       3035000 bytes
+Requests per second:    3328.26 [#/sec] (mean)
+Time per request:       7.511 [ms] (mean)
+Time per request:       0.300 [ms] (mean, across all concurrent requests)
+Transfer rate:          2707.46 [Kbytes/sec] received
+
+Connection Times (ms)
+              min  mean[+/-sd] median   max
+Connect:        0    1   0.3      1       3
+Processing:     1    7  10.2      3     115
+Waiting:        1    6   9.2      3     115
+Total:          2    7  10.1      4     116
+
+Percentage of the requests served within a certain time (ms)
+  50%      4
+  66%      4
+  75%      5
+  80%      9
+  90%     23
+  95%     25
+  98%     27
+  99%     28
+ 100%    116 (longest request)
+```
+
+What do you notice about the profile above? How does it compare to what you see?
+
+We can see from the examples above that there is a bit variance in the performance of the requests. Look at the median and max values for `Total:` — they are quite different! Some of our requests are taking a long time.
+
+Let's see what we can do about that.
+
+> By the way: dealing with this in the way we're about it to is unrealistic: we haven't looked into **why** there is such variance. In the real world we'd definitely do that first.
```

Comment:
Do we also want to talk at all (or reference something) about how localhost load testing is kind of sketchy, and that actually designing _where_ traffic comes from is complex and important?

Reply:
Yep, agree. Thanks